pkg/acsengine/defaults.go

```diff
@@ -83,7 +83,7 @@ var (
 				ImageOffer:     "UbuntuServer",
 				ImageSku:       "16.04-LTS",
 				ImagePublisher: "Canonical",
-				ImageVersion:   "16.04.201701130",
+				ImageVersion:   "16.04.201709190",
```

Comment:
This is specific to German cloud, and you probably want to verify this version exists in Germany, or leave the German cloud change for a separate PR.